lambda_function.py

```diff
@@ -19,9 +19,9 @@ class BaseHTTPResponse:
 
 def get_entity_info_and_return_slack_message(record: dict) -> str:
```

Comment:
Probably worth renaming this as we've changed what the function extracts

Reply:
It's a cloudwatch alarm entity? :grin: I've changed it.